cloud-common/src/main/resources/application.yml

```diff
@@ -136,7 +136,7 @@ cb:
                 CDH 6.2 - Data Science: Apache Spark, Apache Hive, Impala=cdh6-data-science;
                 CDH 6.2 - Data Lake: Hive Metastore=cdh6-shared-services;
                 CDP 1.0 - Data Mart: Apache Spark, Apache Hive, Impala, Hue=cdp-data-mart;
-                CDP 1.0 - Data Engineering: Apache Spark, Apache Livy, Apache Zeppelin, Hue=cdp-data-engineering;
+                CDP 1.0 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie=cdp-data-engineering;
```

Comment:
With the rename, the old template name should be renamed in the database as well. Look for similar SQL scripts at `core/src/main/resources/schema/app`

Reply:
Done